src/main/resources/portal.properties.EXAMPLE

```diff
@@ -268,9 +268,9 @@ ehcache.cache_type=none
 # Enable/Disable quick search (currently in beta, default is false)
 # quick_search.enabled=true
 
-# MatchMiner AWS url
-matchminer.url=
-matchminer.token=
+# MatchMiner setting: Used for matching patients with OncoKB curated trials.
```

Comment:
@victoria34 i think we should just get rid of these from this file for now as it is not a public feature and we don't want to anyone to know about it yet.

Reply:
Removed